### PR TITLE
DEV-1880_move_styles_to_customize_route

### PR DESCRIPTION
**Pull request by gurjittoptal**
Please fill out each section even if it's just with "N/A"

#### Plan? (if this draft/incomplete indicate what you intend to do and how)
This will move Styled under a new route customize 

#### What's this PR do?
Moves styles out of /content into /customize

#### Why are we doing this? How does it help us?
the new designs will need this and this shall make /content less cluttered

#### How should this be manually tested?
go to dashboard and check the following
1 Styles content is no longer found beneath pages content on the “content” tab 
2 “Content” tab is renamed to “Pages” 
3 New tab “Customize” exists in main menu 
4 Styles content is now found on “Customize” page 
5 Functionality (Create, Read, Update, Delete) for both Pages and Styles is maintained 
6 User can still access styles from the page editor, and styles created through the page editor are found on the new “Customize” page 
7 Styles displayed are still based on user permissions 

#### Have automated unit tests been added?
no, cypress tests changed  to handle these 

#### How should this change be communicated to end users?
"N/A"

#### Are there any smells or added technical debt to note?
No

#### Has this been documented? If so, where?
no

#### What are the relevant tickets? Add a link to any relevant ones.
https://news-revenue-hub.atlassian.net/browse/DEV-1880

#### Do any changes need to be made before deployment to production (adding environment variables, for example)?
no

#### Are there next steps? If so, what? Have tickets been opened for them?
no

### CODE REVIEW

File: spa/cypress/integration/07 - pages.spec.js [hunk 1]
```diff
@@ -21,13 +21,13 @@ describe('Donation pages list', () => {
   beforeEach(() => {
     cy.forceLogin(hubAdminUser);
     cy.intercept(getEndpoint(LIST_PAGES), { fixture: 'pages/list-pages-1' }).as('listPages');
-    cy.intercept(getEndpoint(LIST_STYLES), { fixture: 'styles/list-styles-1.json' }).as('listStyles');
+    //cy.intercept(getEndpoint(LIST_STYLES), { fixture: 'styles/list-styles-1.json' }).as('listStyles');
```
Comment (benjaminEwhite) [inline spa/cypress/integration/07 - pages.spec.js:24]: Remove this.

File: spa/cypress/integration/07 - pages.spec.js [hunk 2]
```diff
@@ -21,13 +21,13 @@ describe('Donation pages list', () => {
...
     cy.intercept({ method: 'GET', pathname: getEndpoint(USER) }, { body: hubAdminWithContentFlag });
 
     cy.visit(CONTENT_SLUG);
     cy.url().should('include', CONTENT_SLUG);
     cy.wait('@listPages');
-    cy.wait('@listStyles');
+    //cy.wait('@listStyles');
```
Comment (benjaminEwhite) [inline spa/cypress/integration/07 - pages.spec.js:30]: Remove.

File: spa/src/routes.js [hunk 3]
```diff
@@ -2,6 +2,7 @@ export const LOGIN = '/login';
 export const DASHBOARD_SLUG = '/dashboard';
 export const DONATIONS_SLUG = DASHBOARD_SLUG + '/contributions';
 export const CONTENT_SLUG = DASHBOARD_SLUG + '/content';
+export const STYLES_SLUG = DASHBOARD_SLUG + '/customize';
```
Comment (benjaminEwhite) [inline spa/src/routes.js:5]: Maybe consider calling this var `CUSTOMIZE_SLUG` instead?

**Review by benjaminEwhite — APPROVED** [review]
A couple of commented lines to remove, and one suggestion for a change of var name, but LGTM!

I'm going ahead and approving, but please remove the commented lines, and then no need for me to re-review.

**Review by x110dc — APPROVED** [review]

**Pull request merged**